components/nexus-security/src/main/java/org/sonatype/security/DefaultSecuritySystem.java

```diff
@@ -664,101 +618,11 @@ public void changePassword(String userId, String newPassword)
     eventBus.post(new UserPrincipalsExpired(userId, user.getSource()));
   }
 
-  public void forgotPassword(String userId, String email)
```

Comment:
Is there any plan to replace this functionality? Forgotten password handling is seemingly the only valuable "feature" we're losing with this removal.